packages/bootstrap-vue-next/src/components/BButton/BButton.vue

```diff
@@ -6,21 +6,10 @@
     v-bind="computedAttrs"
     @click="clicked"
   >
-    <div
-      v-if="loadingBoolean"
-      class="btn-loading"
-      :class="{'mode-fill': loadingMode === 'fill', 'mode-inline': loadingMode === 'inline'}"
-    >
-      <slot name="loading">
-        <BSpinner class="btn-spinner" :small="size !== 'lg'" />
-      </slot>
-    </div>
-    <div
-      class="btn-content"
-      :class="{'btn-loading-fill': loadingBoolean && loadingMode === 'fill'}"
-    >
+    <BSpinner v-if="loadingBoolean" class="btn-spinner" :small="size !== 'lg'" />
```

Comment:
```suggestion
   <slot name="loading" v-if="loadingBoolean">
    <BSpinner class="btn-spinner" :small="size !== 'lg'" />
    </slot>
```
Double check my work please

Reply:
Some of the remaining classes may be unnecessary. Perhaps you could look into that as well

Reply:
works fine!

Reply:
https://github.com/bootstrap-vue-next/bootstrap-vue-next/pull/1309#discussion_r1270675270

Btn content and btn spinner might not be useful. Assuming they look similar enough, they van be removed